prize-tool/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+authors = ["Solana Maintainers <maintainers@solana.com>"]
+edition = "2018"
+name = "solana-tds-prize-tool"
+description = "Solana Tour de SOL prize tool"
+version = "0.20.0"
+repository = "https://github.com/solana-labs/solana"
+license = "Apache-2.0"
+homepage = "https://solana.com/tds"
+
+[dependencies]
+clap = "2.33.0"
+log = "0.4.8"
+solana-core = { git = "https://github.com/jstarry/solana", branch = "blocktree-process-opts" }
```

Comment:
crates.io work now?

Reply:
Soon, have to finish up this PR first: https://github.com/solana-labs/solana/pull/6248

Reply:
Issue here: https://github.com/solana-labs/tour-de-sol/issues/16